src/main/java/nl/tudelft/ewi/devhub/server/database/entities/issues/Issue.java

```diff
@@ -0,0 +1,42 @@
+package nl.tudelft.ewi.devhub.server.database.entities.issues;
+
+import java.net.URI;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import nl.tudelft.ewi.devhub.server.database.entities.User;
+
+
+@Data
+@Entity
+@Table(name="issues")
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+public class Issue extends AbstractIssue {
+
+	@NotNull	
+	@Column(name="title", length=128)
+	private String title;
+	
+	@NotNull
```

Comment:
I have answered your question for `title` instead of `description`. Now I think of it, I think a description should be both `nullable` and allow the empty string.